intake_elasticsearch/elasticsearch_table.py

```diff
@@ -60,15 +59,24 @@ def _get_schema(self):
                            npartitions=1,
                            extra_metadata=self.metadata)
 
-    def to_dask(self):
+    def to_dask(self, npartitions=1):
```

Comment:
npartitions should be in the source definition (i.e., argument to `__init__`), not here. That way the API here stays as in the superclass, without parameters.

Reply:
Will do.

Reply:
Yes, exactly. You could call the parameter something different, if you wish, but the source npartition should be updated in any case, this is something that users will want to know.